src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.EasyRequest.cs

```diff
@@ -264,7 +264,8 @@ internal void SetProxyOptions(Uri requestUri)
                 SetCurlOption(CURLoption.CURLOPT_PROXYPORT, proxyUri.Port);
                 VerboseTrace("Set proxy: " + proxyUri.ToString());
 
-                NetworkCredential credentials = GetCredentials(_handler.Proxy.Credentials, _requestMessage.RequestUri);
+                Tuple<NetworkCredential, ulong> credentialScheme = GetCredentials(_handler.Proxy.Credentials, _requestMessage.RequestUri);
```

Comment:
We should use ```KeyValuePair<NetworkCredential, ulong>``` instead of ```Tuple<NetworkCredential, ulong>```; that way we won't allocate a new tuple object for each call to GetCredentials.